recipes/cosmosis-build-standard-library/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "cosmosis-build-standard-library" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:  https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d818aa4f0bebf3d5507014a7042caf1f39636215a348322129647e51fa874055
+
+build:
+  number: 0
+  skip: True # [(py < 36) or win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - cosmosis
+    # For compiling standard librarymodules
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
```

Comment:
I'd probably depend directly on `compilers` here.

Reply:
I tried this today but got a large number of "Overlinking" errors for all the libs that cosmosis supplied, so I've left as-is.  I managed to forget to copy them before closing the terminal - let me know if you'd like me to run again and get them!

Reply:
I'm confused by this. Can you submit the change and we can see what happens in the CI?

Reply:
Done